### ISSUE TITLE
Missing data - subfolders

### ISSUE THREAD

**Issue by simon-donike** [worldstrat/worldstrat#9]
Hello,
after downloading the data from Zenodo and trying to retrain the Benchmarks, I get file not found errors.
`rasterio.errors.RasterioIOError: --path--/lr_dataset/ASMSpotter-1-1-1/L2A/: No such file or directory`
As it turns out, the folder 'ASMSpotter-1-1-1' is not in the dataset even though the download was successful and the checksums were correct. Any ideas how that can happen?

Edit: The above mentioned folder is indeed the only one missing, removing this line form the `stratified_train_val_test_split.csv` file allows the training to continue.

**Comment by jucor** [comment]
Thanks a lot for flagging this! We re downloading the repo to reproduce and fix :) Please bear with us.

**Comment by ivanorsolic** [comment]
Hi @simon-donike, thank you for raising the issue, I confirm I can reproduce it by downloading from Zenodo.
I can also confirm I have located the missing data, and am working on a fix to make it available.

The repo will be getting some updates in the coming weeks, and I will make it a priority to at least provide a downloadable zip with the missing data. After that, I'll work on rebuilding and reupload a patched archive, which takes a bit (lots of data!).

**Comment by nilsleh** [comment]
@ivanorsolic @jucor Is there any update on this? I am also encountering this.

**Comment by jucor** [comment]
Thanks for chasing, @nilsleh ! Since there's only two of us attempting to maintain on the side of now completely unrelated jobs, plus "real life stuff", we're a "little" behind. @ivanorsolic has been working hard on a proper fix that we need to upload (we wanted to do a few more checks before doing so), but there are few non-Worldstrat things that got in the way (morale of the story: push early, even if not done testing?). 
TLDR: We have not forgotten this bug, and we'll push the fix as soon as we can (couple weeks maybe?).

**Comment by nilsleh** [comment]
@jucor just a ping, to see whether there is any update so far? :)

**Comment by jucor** [comment]
Dear all
Good news: we have the fix, and @ivanorsolic and I are planning to upload the missing AOI to Zenodo in the coming days -- just checking that the quota extension we obtained there for the original upload still applies. Will update as soon as uploaded!
